atm.js

```diff
@@ -1,21 +1,73 @@
 var checkingBalance = document.querySelector("div#checking_balance");
 checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
+checkingDeposit = function(amount){
   checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+};
+// var checkingBalance = document.querySelector(".checking_balance").innerHTML;
+var extraAmount = 0;
 
-}
+checkingDeposit = function(){
```

Comment:
You have already defined checkingDeposit above and are reassigning it here, just remember to watch out when naming functions and variables